worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -870,6 +870,45 @@ namespace RTC
 
 		if (isSyncPacket)
 		{
+			if (!lastSentRtpPacketHasMarker && shouldSwitchCurrentSpatialLayer)
```

Comment:
We always use `this->` for class instance members:

```suggestion
			if (!this->lastSentRtpPacketHasMarker && shouldSwitchCurrentSpatialLayer)
```